### ISSUE TITLE
Unauthorized Use of Viacom IP

### ISSUE THREAD

**Issue by mbh-paramount** [lktornado360/nickelodeon#1]
Dear lktornado360,

I am reaching out to you today on behalf of Viacom International Inc. (“Viacom”), subsidiary of Paramount Global, regarding your unauthorized use of Viacom’s NICKELODEON trademarks and related copyrights in this repo, as well as your use of the NICKJR, NOGGIN, and TMNT trademarks and related copyrights in some of your other repos.  Based on your GitHub activity, we understand that you are likely a fan of web-based Flash games by Viacom and others, and are therefore re-posting and/or embedding them and providing others with instructions for how to play them now that most web browsers have ended support for Flash.  However, the .swf game files, trademarks, webpage titles, and subdomains (e.g. nickelodeon.us.to, noggin.us.to, etc.) used in these repos are all Viacom’s intellectual property, which you are using without Viacom’s permission.  Moreover, the website content available at nickelodeon.us.to and noggin.us.to each display Viacom’s intellectual property in a manner that is likely to mislead consumers into believing that the websites are sponsored by or affiliated with Viacom.

We decided to reach out to you directly in advance of filing intellectual property complaints with GitHub because your use of Viacom’s intellectual property appears to stem from your fan interest in Viacom’s web-based Flash games.

At this time, we respectfully request that you remove the following repos based on their unauthorized use of Viacom’s intellectual property:
•	https://github.com/lktornado360/nickelodeon
•	https://github.com/lktornado360/lktornado360.github.io
•	https://github.com/lktornado360/tmnt
•	https://github.com/lktornado360/nickjr

In addition, we request that you delete the subdomains nickelodeon.us.to and noggin.us.to you configured for these websites from the FreeDNS service.

Finally, we request that you cease all other use of Viacom’s intellectual property, and refrain from using any Viacom intellectual property in the future.  If you have any questions, you may reply to this issue, or reach out to me directly at the email address listed in my GitHub profile.

Nothing in this correspondence shall be deemed a waiver of any rights, remedies or defenses of Viacom, its parents, subsidiaries, and affiliates, all of which are expressly reserved.

**Comment by lktornado360** [comment]
Yes sir. I am sorry about using your IP, and I will delete them as soon as
I can.
